udp-ballerina/connectionless_client.bal

```diff
@@ -65,27 +65,34 @@ public client class Client {
     }
 }
 
-# This represent the IP socket address.
-#
-# + host - The hostname of the Socket Address
-# + port - The port number of the Socket Address
-public type Address record {|
-    string host;
-    int port;
-|};
 
 # A self-contained, independent entity of data carrying sufficient information
 # to be routed from the source to the destination nodes without reliance
 # on earlier exchanges between the nodes and the transporting network.
 # 
-# + remoteAddress - The remote address of the remote host
+# + remoteHost - The hostname of the remote host
+# + remotePort - The port number of the remote host
 # + data - The content which needs to be transported to the remote host
 public type Datagram record {|
-   Address remoteAddress;
-   byte[] data;
+   string remoteHost;
+   int remotePort;
+   readonly byte[] data;
 |};
 
-isolated function initConnectionlessClient(Client udpClient, string? host, int timeout) returns Error? =
+#Configurations for the connectionless udp client
+# 
+# + localHost - Local binding of the interface
+# + timeoutInMillis - The socket reading timeout value to be used in milliseconds. If this is not set,
+#                         the default value of 300000 milliseconds (5 minutes) will be used.
+//connectionless client configurations
```

Comment:
Move this comment line to the documentation as it may affect the documentation

Reply:
removed